src/object/C4Shape.cpp

```diff
@@ -25,180 +25,229 @@
 #include "landscape/C4Landscape.h"
 #include "landscape/C4Material.h"
 
+
```

Comment:
We don't use double blank lines anywhere else.

Reply:
Removed them again